jujugui/static/gui/src/app/components/accordion-section/accordion-section.js

```diff
@@ -0,0 +1,110 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+/**
+  A single accordion section, with clickable header and collapsable content.
+  Providing a height will animate the opening and closing.
+
+  XXX: I'd like to remove the requirement for height when animating, but I
+  ran out of time when building this. Luke 12-07-2017.
+*/
+class AccordionSection extends React.Component {
```

Comment:
It's the start of a new pattern that'll be used throughout DF and the profile. The expanding row is also an `li` - which is misused everywhere and is wildly un-semantic in many places around the GUI.